File: 2614-maximum-count-of-positive-integer-and-negative-integer/maximum-count-of-positive-integer-and-negative-integer.java
class Solution {
    public int maximumCount(int[] nums) {
        int i, negCount = 0; 
        for (i = 0; i < nums.length; i++) {
            if (nums[i] < 0) {
                negCount++;
            }
            if (nums[i] > 0) {
                break;
            }
        }
        return Math.max(negCount, (nums.length - i));
    }
}

File: 2614-maximum-count-of-positive-integer-and-negative-integer/maximum-count-of-positive-integer-and-negative-integer.rs
impl Solution {
    pub fn maximum_count(nums: Vec<i32>) -> i32 {
        let mut neg_count = 0;
        let mut i = 0;

        while i < nums.len() {
            if nums[i] < 0 {
                neg_count += 1;
            }
            if nums[i] > 0 {
                break;
            }
            i += 1;
        }

        neg_count.max((nums.len() - i) as i32)
    }
}